telegram/ext/picklepersistence.py

```diff
@@ -50,9 +50,10 @@ class PicklePersistence(BasePersistence[UD, CD, BD]):
 
     .. versionchanged:: 14.0
         The parameters and attributes ``store_*_data`` were replaced by :attr:`store_data`.
+        The parameter and attribute ``filename`` were replaced by :attr:`filepath.
 
     Args:
-        filepath (:obj:`str` | :obj: 'Path'): The filename for storing the pickle files.
+        filepath (:obj:`str` | :obj: 'pathlib.Path'): The filename for storing the pickle files.
```

Comment:
```suggestion
        filepath (:obj:`str` | :obj:`pathlib.Path`): The filename for storing the pickle files.
```

Same below. Otherwise it won't render correctly - you can build the docs locally as described in the contribution guide ;)

Reply:
of course, will do. sorry for all these docs problems, i am new to this style of documentation .

Reply:
No worries :)

Reply:
done